journal/updated/01-journal.adoc

```diff
@@ -767,251 +767,270 @@ nevertheless, afterward it yields the peaceable fruit of righteousness,
+The Lord still commanded me to go, showing that He would provide a help-mate for me.
 And when I had made a covenant with the Lord to go, immediately my pain was removed,
 and I had peace and quietness of mind and spirit.
-I arose next morning, and went to my work; and when those tender Friends,
-that had a regard for me in my sickness, came to see me that morning, I was gone to work;
+I arose the next morning, and went to my work; and when those tender Friends
+who had a regard for me in my sickness came to see me that morning,
+I had gone to work,
 which was to their admiration.
 
-So the Lord gave me a little time, and he alone provided a help-mate for me;
-for I prayed unto him, that she might be of his own providing,
+So the Lord gave me a little time, and He alone provided a help-mate for me;
+for I prayed unto Him that she might be of His own providing,
 for it was not yet manifest to me where she was, or who she was.
-But one time as I was at Horselydown meeting in Southwark,
-I heard a woman Friend open her mouth,
-by way of testimony against an evil ranting spirit that
-did oppose Friends much in those days.
-It came to me from the Lord that that woman was to be my wife,
-and to go with me to the country, and to be a help-mate for me.
+But one time, as I was at Horselydown meeting in Southwark,
+I heard a woman Friend open her mouth
+by way of testimony against an evil, ranting spirit that
+opposed Friends much in those days.
+It was revealed to me by the Lord that that woman was to be my wife,
+and she was to go with me to the country, and be a help-mate to me.
 After meeting, I drew somewhat near to her, but spoke nothing,
-nor took any acquaintance with her, nor did I know when, or where I should see her again.
-I was very willing to let the Lord order it as it seemed best to himself,
-and therein I was easy; and in time the Lord brought us acquainted one with another,
+nor made any acquaintance with her, nor did I know when or where I would see her again.
+I was very willing to let the Lord order it as it seemed best to Himself,
+and therein I had peace of mind; and in time the Lord brought
+us into acquaintance with one another,
 and she confessed that she had some sight of the
 same thing that I had seen concerning her.
 So after some time we parted, and I was freely resigned to the will of God;
-and when we came together again, I told her, if the Lord did order her to be my wife,
+and when we came together again, I told her that if the Lord ordered her to be my wife,
 she must come with me to a strange country,
-where there were no Friends but what God in time might
-call and gather to himself. Upon a little consideration,
-she said, if the Lord should order it so, she must go with her husband,
-though it were to the wilderness;
+where there were no Friends, beside what God in time might
+call and gather to Himself. Upon a little consideration,
+she said that if the Lord ordered it so, she must go with her husband,
```

Comment:
I think this "should" can go back in. ... she said that if the Lord **should order** it so, **then** she must go with her husband